src/components/structural/header/Header.js

```diff
@@ -553,6 +555,43 @@ class Header extends Component {
         this.setState({ snackOpen: false });
     }
 
+    onExport = (scene = undefined) => {
```

Comment:
I noticed that some, but not all functions in the header have brief inline JSDoc-style documentation, should this be added for these new functions?